src/utils/components/GuidedTour/utils/constants.tsx

```diff
@@ -68,7 +68,7 @@ export const tourSteps: Step[] = [
   },
```

Comment:
is it working the same for non admin users?

Reply:
Ah.. this is why all-namespaces right ?

Reply:
okay now fixed. all-namespaces url are all supported and the VM link is 
/k8s/ns/default/kubevirt.io~v1~VirtualMachine/rhel9-tour-guide/configuration
but because of my change in the [VirtualMachineNavPage.tsx](https://github.com/kubevirt-ui/kubevirt-plugin/pull/2475/files#diff-e4680557f12a577766937e346ef2012e2014529efa4fa62c429ba629ac82cfc3) , everything works for the non-admin user